sources/packages/backend/apps/db-migrations/src/sql/DisbursementSchedules/Add-has-estimated-awards-col.sql

```diff
@@ -0,0 +1,35 @@
+-- Set the default to false initially
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN has_estimated_awards BOOLEAN NOT NULL DEFAULT FALSE;
+
+COMMENT ON COLUMN sims.disbursement_schedules.has_estimated_awards IS 'Indication for whether the disbursement has estimated awards greater than $0.';
+
+-- Update query to calculate TRUE | FALSE based on the data.
+UPDATE
+    sims.disbursement_schedules
+SET
+    has_estimated_awards = CASE
+        WHEN disbursement_total_amounts.value_amount = 0 THEN false
+        ELSE true
+    END
+FROM
+    (
+        SELECT
+            disbursement_schedules.id AS disbursement_schedules_id,
+            sum(disbursement_values.value_amount) AS value_amount
+        FROM
+            sims.disbursement_schedules disbursement_schedules
+            INNER JOIN sims.disbursement_values disbursement_values ON disbursement_values.disbursement_schedule_id = disbursement_schedules.id
+        GROUP BY
+            disbursement_schedules.id
+    ) disbursement_total_amounts

```

Comment:
As a suggestion, I would recommend the following SQL considering less complex to read and understand IMO. 

But it is not a blocker. Existing update still does the work. 

```sql
update
	sims.disbursement_schedules
set
	has_estimated_awards = (case
		when exists (
	select
			1
	from
			sims.disbursement_values disbursement_value
	where
			disbursement_value.disbursement_schedule_id = sims.disbursement_schedules.id
		and disbursement_value.value_amount > 0) then true
	else false
end);
```

Reply:
Thanks for the query. I updated this query in this [commit](https://github.com/bcgov/SIMS/pull/4010/commits/f7ba06962446e5a7ae2b04f62ccf9f3f3a47a714), but sonarcloud quality gate failed.
![image](https://github.com/user-attachments/assets/fb674046-c240-4484-94e5-17b752526604)
![image](https://github.com/user-attachments/assets/aaafb5f0-85c2-411a-9126-91d81854e52e)
![image](https://github.com/user-attachments/assets/cc76f075-8669-4acf-8f2f-50eae7bade5d)
I am reverting it back with the update in this [comment](https://github.com/bcgov/SIMS/pull/4010#discussion_r1866932371).